keras/layers/wrappers.py

```diff
@@ -174,7 +203,11 @@ def step(x, _):
                 input_length = K.shape(inputs)[1]
             # Shape: (num_samples * timesteps, ...)
             inputs = K.reshape(inputs, (-1,) + input_shape[2:])
-            y = self.layer.call(inputs)  # (num_samples * timesteps, ...)
+            reshaped_mask = self._reshape_input_mask(mask)
+            try:
+                y = self.layer.call(inputs, reshaped_mask)  # supports mask
+            except TypeError:
```

Comment:
It will not mess up the op as the `try` expression will fail if `self.layer.call` only accept `inputs` (e.g., `Dense`), so no expression in the function is ever executed.